simulation/sub8_gazebo/launch/duck.launch

```diff
@@ -28,6 +28,18 @@
       -x 23 -y 11 -z -4.72 -Z 1.5707 -model torpedo_board"
     output="screen" />
 
+  <!-- Attempting to spawn the bin and lid -->
+  <node
+    name ="bin_assembly" pkg="gazebo_ros" type="spawn_model"
+    args="-file $(find sub8_gazebo)/models/bin/bin_assembly.sdf -sdf
+    -x -22.0 -y 14.0 -z -4.52 -Z -1.57 -model bin_assembly"
+    output="screen" />
+  <node
+    name ="lid" pkg="gazebo_ros" type="spawn_model"
+    args="-file $(find sub8_gazebo)/models/bin/lid.sdf -sdf
+    -x -22.3 -y 14.16 -z -4.47 -Z -1.57 -model lid"
+    output="screen" />
```

Comment:
It should be easy to add gravity to the lid later since it is separate from the bin. We will do this when we are ready to attempt that mission since there are more important things to get done in the simulator right now.